app/views/external_users/claims/fixed_fees/_fields.html.haml

```diff
@@ -1,5 +1,5 @@
 - if @claim.allows_fixed_fees?
-  #fixed-fees.mod-fees{"data-mute-indictment": @claim.allows_fixed_fees? ? "true" : "false"}
+  #fixed-fees.mod-fees.fx-numberedList-wrapper
```

Comment:
`fx-numberedList-wrapper` is this part of this PR?

Reply:
It is not.. :)